controllers/webhook/ksvc_validator.go

```diff
@@ -26,11 +27,11 @@ func NewKsvcValidator(client client.Client) *ksvcValidator {
 	return &ksvcValidator{client: client}
 }
 
-func (v *ksvcValidator) ValidateCreate(ctx context.Context, obj runtime.Object) error {
+func (v *ksvcValidator) ValidateCreate(ctx context.Context, obj runtime.Object) (warnings admission.Warnings, err error) {
```

Comment:
Why do you need to add an additional return object (warnings) that seems to be always null?

Reply:
its a new method signature of `ValidateCreate` in controller-runtime v0.16.3. 
https://github.com/kubernetes-sigs/controller-runtime/blob/v0.16.4/pkg/webhook/admission/validator.go#L42